core/util_test.go

```diff
@@ -347,6 +351,7 @@ func (d *DrandTest2) RunReshare(oldRun, newRun, newThr int, timeout time.Duratio
 		require.NoError(d.t, err)
 		_, err = client.InitReshare(leader.drand.priv.Public, secret, d.groupPath, force)
 		if err != nil {
+			fmt.Println("error in NON LEADER: ", err)
```

Comment:
printlns should be turned into logs or removed before merge :)

Reply:
wait until I mark it as ready before reviewing !! :D